README.md

```diff
@@ -65,7 +65,7 @@ Currently, the following games are supported:
 * Castlevania 64
 * A Short Hike
 * Yoshi's Island
-
+* Mario & Luigi: Superstar Saga
```

Comment:
I can't believe I removed the *line* and not just the bullet. My bad
```suggestion
* Mario & Luigi: Superstar Saga

```

Reply:
smh